src/test/java/DukeTest.java

```diff
@@ -0,0 +1,26 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import duke.Deadline;
+import duke.Event;
+import duke.Task;
+
+public class DukeTest {
+    @Test
+    public void dummyTest() {
+        assertEquals(2, 2);
+    }
+
+    @Test
+    public void test1() {
```

Comment:
Maybe you can name your tests according to the featureUnderTest_testScenario_expectedBehavior() format?